blimp-live-widgets/src/components/DateComponent/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import styles from './styles.css'
+import { updateDate } from './utils';
+
+export default class DateComponent extends Component {
+
+  
+  constructor(props) {
+    var currentDate = new Date()
+    super(props);
+    
+    const date = this.initializeDate(props.gmtOffset);
+    this.state = {
+      weekday:date[0],
+      month:date[1],
+      day: date[2],
+      year: date[3],
+    };
+  }
+
+  componentDidMount() {
+    this.interval = setInterval(() => {
+      this.setState(updateDate(this.state));
+    console.log(this.state)
```

Comment:
Please remove and also since ur removing this, on lines 16-19 can u make the spacing more consistent.